dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -0,0 +1,46 @@
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!")})
+  .catch((err) => {
+    console.log("Failed to connect Database", err)
+    appDataSource.destroy();
+  });
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+
+//health check
+app.get("/ping", (req, res) => {
+return res.status(200).json({ message: "pong" });
```

Comment:
사소하지만 가독성을 위해 들여쓰기 (indentation) 한번 해주세요~!

```suggestion
  return res.status(200).json({ message: "pong" });
```